.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: "Deploy site"
+
+on:
+  pull_request:
+    branches:
+    - '*'
+  push:
+    branches:
+    - 'master'
+
+jobs:
+  example-1:
+    name: Deploy lektor website
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
```

Comment:
actions/setup-node@v2 is available (though I have no idea how it differs from @v1...)